README.markdown

```diff
@@ -62,6 +62,7 @@ http {
 
             wait_interval = 0.010,  -- wait before retry fetching event data
             wait_max = 0.5,         -- max wait time before discarding event
+            shm_retry = 5,          -- retries for shm fragmentation (no memory)
```

Comment:
Could we rename to `shm_retries` instead? `shm_retry` sounds like a boolean (retry? yes / no)